### ISSUE TITLE
Manual Update Check Fails

### ISSUE THREAD

**Issue by AntumDeluge** [bjorn/tiled#2003]
System: **Windows 10 64-bit**
Tiled version: **1.1.6 64-bit (installed via MSI)**

*(Issue occurred for me with Tiled versions **1.1.5** & **1.1.6**)*

Went to ***Edit -> Preferences... -> Updates -> Check Now***. After a short progress dialog, the following error was displayed:
![update_error](https://user-images.githubusercontent.com/3631473/45132393-a4c47b80-b145-11e8-8d10-e0deec7f527d.png)

Confirmed that I am connected to the internet.

**Comment by bjorn** [comment]
Yeah, sorry about that. So far I was renewing the SSL certificates manually, but now I am in the process of moving to a new server setup that auto-renews those certificates. However, I did not get around to moving `update.mapeditor.org` over to the new host before its current certificate expired.

I expect to have time for this latest on Tuesday next week. In the meantime, at least it only pops up an error when you manually check for updates.

**Comment by AntumDeluge** [comment]
Okay, glad to know that it's not a big deal & will be fixed. :)

I'll leave this open for now (unless you want to close it) for others to see so they will avoid creating a new report.

**Comment by oktonion** [comment]
@bjorn situation is the same for me with 1.2.0.

**Comment by bjorn** [comment]
@oktonion What, you get an Update Error message? Can you access https://update.mapeditor.org/ without warnings?

**Comment by oktonion** [comment]
> 
> 
> @oktonion What, you get an Update Error message? Can you access https://update.mapeditor.org/ without warnings?

Yep. From browser it works but with proxy only. Seems it is something with traffic routes. It could be nice for app to use proxy too if update is not accessible tho. But I understand that this is not no.1 priority.

**Comment by oktonion** [comment]
@bjorn You can use [this pac](https://antizapret.prostovpn.org/proxy.pac). It is updated every day and is open and hosted on GitHub. Or any other proxy will be great.

**Comment by AntumDeluge** [comment]
Just wanted to report that update from 1.1.6 to 1.2.1 went smoothly. `Edit -> Preferences -> Updates -> Check Now` reports correctly. Thank you for your work.

**Comment by bjorn** [comment]
> @bjorn You can use [this pac](https://antizapret.prostovpn.org/proxy.pac). It is updated every day and is open and hosted on GitHub. Or any other proxy would be great.

I don't understand. What would I do with that pac? I thought you wanted Tiled to pick up your system proxy settings when accessing the internet (I'm not sure if it does).

> Just wanted to report that update from 1.1.6 to 1.2.1 went smoothly. `Edit -> Preferences -> Updates -> Check Now` reports correctly. Thank you for your work.

@AntumDeluge Yeah, when I released 1.2.1 I also finally updated the files that Tiled uses to see if there's a new version. Glad to hear the process worked fine!
